### ISSUE TITLE
trailing space in Saint Pierre and Miquelon nationalNumberPattern

### ISSUE THREAD

**Issue by marmelroy** [googlei18n/libphonenumber#1039]
```   <!-- Saint Pierre and Miquelon (Collectivité territoriale de la République française) -->
    <territory id="PM" countryCode="508" internationalPrefix="00"
               nationalPrefix="0" nationalPrefixFormattingRule="$NP$FG">
      <references>
        <sourceUrl>http://www.itu.int/oth/T02020000B2/en</sourceUrl>
      </references>
      <availableFormats>
        <numberFormat pattern="([45]\d)(\d{2})(\d{2})">
          <format>$1 $2 $3</format>
        </numberFormat>
      </availableFormats>
      <generalDesc>
        <nationalNumberPattern>[45]\d{5}</nationalNumberPattern>
        <possibleNumberPattern>\d{6}</possibleNumberPattern>
      </generalDesc>
      <fixedLine>
        <nationalNumberPattern>41\d{4} </nationalNumberPattern>
        <exampleNumber>411234</exampleNumber>
      </fixedLine>
      <mobile>
        <nationalNumberPattern>55\d{4} </nationalNumberPattern>
        <exampleNumber>551234</exampleNumber>
      </mobile>
    </territory>```

I believe this is bug.

**Comment by padmaksha** [comment]
Hi Roy,

Thanks for providing all the details we need!
We've accepted this issue and will fix this in an upcoming release.

**Comment by penmetsaa** [comment]
This is fixed in release 7.3.2.
